ofrak_core/Dockerstub

```diff
@@ -27,6 +27,21 @@ RUN wget https://raw.githubusercontent.com/iBotPeaches/Apktool/v2.3.3/scripts/li
   wget https://github.com/patrickfav/uber-apk-signer/releases/download/v1.0.0/uber-apk-signer-1.0.0.jar -O /usr/local/bin/uber-apk-signer.jar && \
   chmod +x /usr/local/bin/apktool*
 
+# Install official 7-zip
+RUN if [ "$TARGETARCH" = "arm64" ]; then \
+  wget https://www.7-zip.org/a/7z2201-linux-arm64.tar.xz && \
+  tar -xf 7z2201-linux-arm64.tar.xz && \
+  mv 7zz /usr/local/bin && \
+  rm 7z2201-linux-arm64.tar.xz 7zzs; \
+fi;
+
+RUN if [ "$TARGETARCH" = "amd64" ]; then \
+  wget https://www.7-zip.org/a/7z2201-linux-x64.tar.xz && \
+  tar -xf 7z2201-linux-x64.tar.xz && \
+  mv 7zz /usr/local/bin && \
+  rm 7z2201-linux-x64.tar.xz 7zzs; \
+fi;
```

Comment:
I see that these Docker steps clean up after themselves with the `rm` commands, but this should still be run from the `/tmp` directory in case we update the tarball URLs in the future, and they unpack files besides just `7zz` and `7zzs`. In other words, we want to make sure not to leave garbage in whatever the working directory is at the time this runs.

```suggestion
# Install official 7-zip
RUN if [ "$TARGETARCH" = "arm64" ]; then \
  cd tmp && \
  wget https://www.7-zip.org/a/7z2201-linux-arm64.tar.xz && \
  tar -xf 7z2201-linux-arm64.tar.xz && \
  mv 7zz /usr/local/bin && \
  rm 7z2201-linux-arm64.tar.xz 7zzs; \
fi;

RUN if [ "$TARGETARCH" = "amd64" ]; then \
  cd tmp && \
  wget https://www.7-zip.org/a/7z2201-linux-x64.tar.xz && \
  tar -xf 7z2201-linux-x64.tar.xz && \
  mv 7zz /usr/local/bin && \
  rm 7z2201-linux-x64.tar.xz 7zzs; \
fi;
```